### PR TITLE
Update alpha warning from docs

### PR DESCRIPTION
**Pull request by srt32**
I'm not totally sure we're ready to remove this warning but given we have various components in beta, I think it's time to discuss!

### CODE REVIEW

**Comment by joelhawksley** [comment]
@srt32 I think you're right to question why we still have this notice.

I still think we should have _some_ sort of disclaimer, but perhaps we can soften the current one:

_Note: This library is under active pre-1.0 development. Breaking changes are likely in patch releases._

File: README.md [hunk 1]
```diff
@@ -6,8 +6,6 @@
 
 <p align="center">ViewComponents for the Primer Design System</p>
```
Comment (srt32) [inline README.md:8]:
```suggestion

_Note: This library is under active pre-1.0 development. Breaking changes are likely in patch releases._

```

File: docs/content/index.md [hunk 2]
```diff
@@ -7,8 +7,6 @@ export default HeroLayout
 
 Primer ViewComponents is an implementation of the Primer Design System, using [ViewComponent](https://github.com/github/view_component).
```
Comment (srt32) [inline docs/content/index.md:9]:
```suggestion
_Note: This library is under active pre-1.0 development. Breaking changes are likely in patch releases._


```

**Comment by srt32** [comment]
@joelhawksley I incorporated your better language. Good to go?

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**